libphx/src/shader.rs

```diff
@@ -68,13 +70,19 @@ unsafe fn CreateGLShader(src: *const libc::c_char, type_0: gl::types::GLenum) ->
 
     /* Check for compile errors. */
     let mut status: i32 = 0;
+
```

Comment:
This spacing makes it harder to read IMO.

Reply:
Removed. But for me it's other way around :smile: